graphql/util.go

```diff
@@ -58,40 +58,77 @@ func getGroupedFiles(ctx context.Context, name string, taskID string, execution
 	return &GroupedFiles{TaskName: &name, Files: apiFileList}, nil
 }
 
-func setScheduled(ctx context.Context, url, taskID string, isActive bool) (*restModel.APITask, error) {
+func setManyTasksScheduled(ctx context.Context, url string, isActive bool, taskIDs ...string) ([]*restModel.APITask, error) {
 	usr := mustHaveUser(ctx)
-	t, err := task.FindOneId(taskID)
+	tasks, err := task.FindAll(db.Query(task.ByIds(taskIDs)))
 	if err != nil {
 		return nil, ResourceNotFound.Send(ctx, err.Error())
 	}
-	if t == nil {
-		return nil, ResourceNotFound.Send(ctx, errors.Errorf("task %s not found", taskID).Error())
+	if len(tasks) != len(taskIDs) {
+		foundTaskIds := []string{}
+		for _, ft := range tasks {
+			foundTaskIds = append(foundTaskIds, ft.Id)
+		}
+		missingTaskIds := sliceDifference(taskIDs, foundTaskIds)
+		grip.Error(message.Fields{
+			"message":      "could not find all tasks",
+			"missingTasks": missingTaskIds,
+		})
+		return nil, ResourceNotFound.Send(ctx, errors.New("tasks not found").Error())
```

Comment:
Should we return here and still modify the tasks that we do have?

Reply:
Sure, I suppose we can remove the return statement here and just let the grip.error remain.

Reply:
Actually I might still error if we find no tasks at all because otherwise the output will be '"status": "started"' which isn't accurate.